### ISSUE TITLE
[BUG]: live-painting: sometimes the output image is broken

### ISSUE THREAD

**Issue by TimPietrusky** [blib-la/captain#64]
**Describe the bug** 
In very rare cases, the output image is broken for one frame. 

**Expected behavior**
The output is never broken. 

- OS: Windows 11
- Captain: 1.0.0-alpha.5

**Comment by pixelass** [comment]
This is the error when a glitched image appears

```
live-painting: {"status":"image_generated"}
Received non-JSON data: Failed to rename C:\...\temp\live-painting\output.png.tmp.png to
C:\...\temp\live-painting\output.png after 5 attempts.
Error: [WinError 2] The system cannot find the file specified: 'C:\\...\\temp\\live-painting\\output.png.tmp.png' -> 'C:\\...\\temp\\live-painting\\output.png'
{"status": "image_generated"}
```

**Comment by pixelass** [comment]
fixed by #282
